packages/server/src/items/smashable.ts

```diff
@@ -19,9 +19,12 @@ export class Smashable {
 
   private mob!: Mob; // Add mob as a class property
 
-  smashItem(mob: Mob) {
+  // default param for testability, only call with one param in normal use
```

Comment:
I recommend using a JSDoc for this, like so:
```
/**
 * Add 2 numbers
 * @param a The first number
 * @param b The second number
 * @returns The sum of both numbers
 */
function sum(a: number, b: number) {
  return a + b;
}
```

Then we get nice documentation when hovering over it.

Here's a nice paragraph from Google describing the difference between line comments and JSdocs [https://google.github.io/styleguide/tsguide.html#jsdoc-vs-comments](https://google.github.io/styleguide/tsguide.html#jsdoc-vs-comments)